### PR TITLE
Config paper

### PR DESCRIPTION
**Pull request by susuhahnml**

### CODE REVIEW

File: krr.bib [hunk 1]
```diff
@@ -6645,6 +6645,18 @@ @inproceedings{coevgeprscsith13a
   pages = {245-256}
 }
 
+@inproceedings{cofahascsc23a,
+  title = {Interactive Configuration with {ASP} Multi-Shot Solving},
+  author = {R. Comploi{-}Taupe and
+A. Falkner and
+S. Hahn and
+T. Schaub and
+G. Schenner},
```
Comment (tortinator) [inline krr.bib:6654]:
```suggestion
  author = {R. Comploi{-}Taupe and A. Falkner and S. Hahn and T. Schaub and G. Schenner},
```

File: procs.bib [hunk 2]
```diff
@@ -897,6 +897,18 @@ @proceedings{conf18
   year = {2018}
 }
 
+@proceedings{conf23,
+  title = {Proceedings of the Twenty-fifth International Configuration Workshop (CONF'25)},
+  editor = {J. Horcas and
+J. Galindo and
+R. Comploi{-}Taupe and
+L. Fuentes},
```
Comment (tortinator) [inline procs.bib:905]:
```suggestion
  editor = {J. Horcas and J. Galindo and R. Comploi{-}Taupe and L. Fuentes},
```

**Review by tortinator — APPROVED** [review]
Hola, please put auothors/editors on one line :)

**Comment by tortinator** [comment]
Hola @susuhahnml , just check the above changes, committ them and merge :)

**Comment by susuhahnml** [comment]
Done, sorry it took me so long to get back to it @tortinator . I can't merge tho... it is restricted.

**Pull request merged**